src/menu/view/checkable-item.jsx

```diff
@@ -39,6 +40,9 @@ export default class CheckableItem extends Component {
         super(props);
 
         bindCtx(this, ['stopPropagation', 'handleKeyDown', 'handleClick']);
+        this.state = {
```

Comment:
Not quit following, why should it be in state instread of plain private property of the class, like "this.id"

Reply:
i can set it that way

Reply:
changed